gennav/utils/custom_exceptions.py

```diff
@@ -0,0 +1,99 @@
+class Error(Exception):
```

Comment:
Is this neccessary, cant they all inherit `Exception`  directly

Reply:
Not necessary. But in gen, almost in all custom exceptions examples (including the python docs), there's a base Error class similar to this.

Reply:
Yeah, it might be useful if we abstract things into the base class. For now you can remove it

Reply:
Okay.